Modules/ITS/include/ITS/ITSDecodingErrorCheck.h

```diff
@@ -41,7 +41,8 @@ class ITSDecodingErrorCheck : public o2::quality_control::checker::CheckInterfac
   // Override interface
   Quality check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap) override;
   void beautify(std::shared_ptr<MonitorObject> mo, Quality checkResult = Quality::Null) override;
-  std::vector<int> vDecErrorLimits, vListErrorIdBad, vListErrorIdMedium;
+  std::vector<int> vDecErrorLimits, vDecErrorType, vListErrorIdBad, vListErrorIdMedium;
```

Comment:
In principle, we don't need some of these vectors to be defined here, since we create `vDecErrorLimits`, `vDecErrorLimitsRatio`, `vDecErrorType` in the .cxx file